common/build_number.py

```diff
@@ -71,40 +71,43 @@ def increase_build_number(origin_repo_path, os_type, branch):
     build_numbers_file = 'build_numbers.json'
 
     log.info(f'Get build number from local repository')
-    origin_build_number = get_build_number(repo_path=pathlib.Path(origin_repo_path),
+    current_build_number = get_build_number(repo_path=pathlib.Path(local_repo_path),
                                            os_type=os_type, branch=branch)
-    if origin_build_number == 0:
-        log.error(f'Build number should be not 0'
-                  f'Maybe {pathlib.Path(origin_repo_path) / build_numbers_file} do not contains {branch} branch for {os_type} platform')
+    if current_build_number == 0:
+        log.error(f'Local build number must not be 0'
+                  f'Check that {pathlib.Path(local_repo_path) / build_numbers_file} contains appropriate {branch} branch for {os_type} platform')
         return False
 
-    temporary_dir = pathlib.Path(origin_repo_path) / '..' / 'tmp'
-    temporary_repo_path = temporary_dir / 'mdp_msdk-product-configs'
-    temporary_build_number_path = temporary_repo_path / build_numbers_file
+    repo_name = pathlib.Path(local_repo_path).name
+    temp_dir = (pathlib.Path(local_repo_path) / '..' / 'tmp').resolve()
+    latest_version_repo_path = temp_dir / repo_name
+    latest_build_number_path = latest_version_repo_path / build_numbers_file
 
-    if temporary_dir.exists():
-        log.info(f"Remove old repository in {temporary_dir}")
-        remove_directory(str(temporary_dir))
+    if temp_dir.exists():
+        log.info(f"Remove old repository in {temp_dir}")
+        remove_directory(str(temp_dir))
 
-    temporary_dir.mkdir(exist_ok=True)
-    extract_repo(root_repo_dir=temporary_dir, repo_name='mdp_msdk-product-configs',
+    temp_dir.mkdir(exist_ok=True)
+    extract_repo(root_repo_dir=temp_dir, repo_name=repo_name,
                  branch=branch, commit_id='HEAD')
 
-    log.info(f'Getting build number from HEAD of {branch} branch for repo in {temporary_repo_path}')
-    temporary_build_number = get_build_number(repo_path=temporary_repo_path,
-                                              os_type=os_type, branch=branch)
+    log.info(
+        f'Getting build number from HEAD of {branch} branch for repo in {latest_version_repo_path}')
+    latest_build_number = get_build_number(repo_path=latest_build_number_path,
```

Comment:
latest_build_number is not clear. I recommended:
git_build_number or latest_git_build_number